nni/tools/nnictl/nnictl_utils.py

```diff
@@ -202,8 +201,9 @@ def convert_time_stamp_to_date(content):
 
 def check_rest(args):
     '''check if restful server is running'''
-    nni_config = Config(get_config_filename(args))
-    rest_port = nni_config.get_config('restServerPort')
+    experiment_config = Experiments()
+    experiment_dict = experiment_config.get_all_experiments()
+    rest_port = experiment_dict.get(get_config_filename(args), None).get('port', None)
```

Comment:
you put `None` here means `None` may be returned, but None does not have attribute `get`

Reply:
In fact `get_config_filename()` already has check if `get_config_filename(args)` is in `experiment_dict`, so if the value is `None`, there must be an unknown bug. And `{}.get('something', None)` is equal to `{}.get('something')`.

Reply:
got it. then suggest to remove `None` from the first `get`

Reply:
ok, I will move all `None` in `{}.get()`